diff --git a/node_modules/@ember/test-helpers/addon-test-support/@ember/test-helpers/-internal/build-registry.js b/node_modules/@ember/test-helpers/addon-test-support/@ember/test-helpers/-internal/build-registry.js
index 3d33ddc..544598c 100644
--- a/node_modules/@ember/test-helpers/addon-test-support/@ember/test-helpers/-internal/build-registry.js
+++ b/node_modules/@ember/test-helpers/addon-test-support/@ember/test-helpers/-internal/build-registry.js
@@ -1,7 +1,6 @@
 import ApplicationInstance from '@ember/application/instance';
 import Application from '@ember/application';
 import EmberObject from '@ember/object';
-import require, { has } from 'require';
 import Ember from 'ember';
 
 /**
@@ -84,15 +83,6 @@ export default function (resolver) {
   });
   owner.__container__ = container;
   exposeRegistryMethodsWithoutDeprecations(container);
-  if (has('ember-data/setup-container')) {
-    // ember-data is a proper ember-cli addon since 2.3; if no 'import
-    // 'ember-data'' is present somewhere in the tests, there is also no `DS`
-    // available on the globalContext and hence ember-data wouldn't be setup
-    // correctly for the tests; that's why we import and call setupContainer
-    // here; also see https://github.com/emberjs/data/issues/4071 for context
-    let setupContainer = require('ember-data/setup-container')['default'];
-    setupContainer(registry || container);
-  }
   return {
     registry,
     container,
diff --git a/node_modules/@ember/test-helpers/addon-test-support/@ember/test-helpers/settled.js b/node_modules/@ember/test-helpers/addon-test-support/@ember/test-helpers/settled.js
index 02c8c65..9858066 100644
--- a/node_modules/@ember/test-helpers/addon-test-support/@ember/test-helpers/settled.js
+++ b/node_modules/@ember/test-helpers/addon-test-support/@ember/test-helpers/settled.js
@@ -143,6 +143,7 @@ if (loader.registry['ember-testing/test/waiters']) {
   @returns {boolean} true if waiters are still pending
 */
 function checkWaiters() {
+  return false;
   let EmberTest = Ember.Test;
   if (_internalCheckWaiters) {
     return _internalCheckWaiters();
