diff --git a/node_modules/@glimmer/manager/dist/modules/es2017/lib/public/template.js b/node_modules/@glimmer/manager/dist/modules/es2017/lib/public/template.js
index 7cb4d1e..1f00eb2 100644
--- a/node_modules/@glimmer/manager/dist/modules/es2017/lib/public/template.js
+++ b/node_modules/@glimmer/manager/dist/modules/es2017/lib/public/template.js
@@ -2,7 +2,10 @@ import { DEBUG } from '@glimmer/env';
 import { debugToString } from '@glimmer/util';
 const TEMPLATES = new WeakMap();
 const getPrototypeOf = Object.getPrototypeOf;
-export function setComponentTemplate(factory, obj) {
+export function setComponentTemplate(_factory, obj) {
+
+  const factory = getComponentTemplate(_factory) || _factory;
+
   if (DEBUG && !(obj !== null && (typeof obj === 'object' || typeof obj === 'function'))) {
     throw new Error(`Cannot call \`setComponentTemplate\` on \`${debugToString(obj)}\``);
   }
@@ -15,6 +18,11 @@ export function setComponentTemplate(factory, obj) {
   return obj;
 }
 export function getComponentTemplate(obj) {
+
+  if (!TEMPLATES.has(obj) && 'template' in obj) {
+    setComponentTemplate(getComponentTemplate(obj.template) || obj.template, obj);
+  }
+
   let pointer = obj;
 
   while (pointer !== null) {
