diff --git a/node_modules/@glimmer/component/addon/-private/ember-component-manager.ts b/node_modules/@glimmer/component/addon/-private/ember-component-manager.ts
index 3eae1af..2eb765f 100644
--- a/node_modules/@glimmer/component/addon/-private/ember-component-manager.ts
+++ b/node_modules/@glimmer/component/addon/-private/ember-component-manager.ts
@@ -4,15 +4,14 @@ import { set } from '@ember/object';
 import { getOwner, setOwner } from '@ember/application';
 import { capabilities } from '@ember/component';
 import { schedule } from '@ember/runloop';
-import { gte } from 'ember-compatibility-helpers';
 import BaseComponentManager, {
   ComponentManagerArgs,
   CustomComponentCapabilities,
 } from './base-component-manager';
-
 import GlimmerComponent, { Constructor } from './component';
 import * as destroyables from './destroyables';
 const { setDestroyed, setDestroying } = destroyables;
+const gte = () => true;
 
 const CAPABILITIES = gte('3.13.0-beta.1')
   ? capabilities('3.13', {
