diff --git a/node_modules/ember-source/dist/dependencies/@glimmer/manager.js b/node_modules/ember-source/dist/dependencies/@glimmer/manager.js
index f1ed636..ee96189 100644
--- a/node_modules/ember-source/dist/dependencies/@glimmer/manager.js
+++ b/node_modules/ember-source/dist/dependencies/@glimmer/manager.js
@@ -888,4 +888,7 @@ function getComponentTemplate(obj) {
   return undefined;
 }
 
-export { setInternalHelperManager, setInternalModifierManager, setInternalComponentManager, getInternalHelperManager, getInternalModifierManager, getInternalComponentManager, hasInternalHelperManager, hasInternalModifierManager, hasInternalComponentManager, setHelperManager, setModifierManager, setComponentManager, componentCapabilities, CustomComponentManager, modifierCapabilities, CustomModifierManager, helperCapabilities, hasDestroyable, hasValue, CustomHelperManager, getComponentTemplate, setComponentTemplate, capabilityFlagsFrom, hasCapability, managerHasCapability, getCustomTagFor, setCustomTagFor };
+var capabilities = () => CAPABILITIES;
+
+
+export { capabilities, setInternalHelperManager, setInternalModifierManager, setInternalComponentManager, getInternalHelperManager, getInternalModifierManager, getInternalComponentManager, hasInternalHelperManager, hasInternalModifierManager, hasInternalComponentManager, setHelperManager, setModifierManager, setComponentManager, componentCapabilities, CustomComponentManager, modifierCapabilities, CustomModifierManager, helperCapabilities, hasDestroyable, hasValue, CustomHelperManager, getComponentTemplate, setComponentTemplate, capabilityFlagsFrom, hasCapability, managerHasCapability, getCustomTagFor, setCustomTagFor };
diff --git a/node_modules/ember-source/dist/packages/@ember/debug/index.js b/node_modules/ember-source/dist/packages/@ember/debug/index.js
index 681b1a6..ce8bcdd 100644
--- a/node_modules/ember-source/dist/packages/@ember/debug/index.js
+++ b/node_modules/ember-source/dist/packages/@ember/debug/index.js
@@ -275,4 +275,10 @@ if (DEBUG && !isTesting()) {
   }
 }
 
+if (typeof deprecate !== 'function') {
+  deprecate = function (message, test, options) {
+   console.log('deprecate', message, test, options);
+  };
+}
+
 export { assert, info, warn, debug, deprecate, debugSeal, debugFreeze, runInDebug, deprecateFunc, setDebugFunction, getDebugFunction, _warnIfUsingStrippedFeatureFlags };
\ No newline at end of file
diff --git a/node_modules/ember-source/dist/packages/@ember/routing/route.js b/node_modules/ember-source/dist/packages/@ember/routing/route.js
index cead465..927f282 100644
--- a/node_modules/ember-source/dist/packages/@ember/routing/route.js
+++ b/node_modules/ember-source/dist/packages/@ember/routing/route.js
@@ -6,6 +6,7 @@ var __decorate = this && this.__decorate || function (decorators, target, key, d
   return c > 3 && r && Object.defineProperty(target, key, r), r;
 };
 
+import { getComponentTemplate } from '@glimmer/manager';
 import { privatize as P } from '@ember/-internals/container';
 import { addObserver, defineProperty, descriptorForProperty, flushAsyncObservers } from '@ember/-internals/metal';
 import { getOwner } from '@ember/-internals/owner';
@@ -1351,6 +1352,9 @@ function buildRenderOptions(route, nameOrOptions, options) {
   }
 
   let template = owner.lookup(`template:${templateName}`);
+  if (typeof template !== 'finction' && typeof template === 'object') {
+    template = getComponentTemplate(template);
+  }
   assert(`Could not find "${templateName}" template, view, or component.`, isDefaultRender || template !== undefined);
   let parent;
 
diff --git a/node_modules/ember-source/dist/packages/@ember/test/index.js b/node_modules/ember-source/dist/packages/@ember/test/index.js
index 8d78b05..9d15054 100644
--- a/node_modules/ember-source/dist/packages/@ember/test/index.js
+++ b/node_modules/ember-source/dist/packages/@ember/test/index.js
@@ -1,14 +1,15 @@
-import require, { has } from 'require';
+import * as testing from 'ember-testing';
+import { DEBUG } from '@glimmer/env';
 export let registerAsyncHelper;
 export let registerHelper;
 export let registerWaiter;
 export let unregisterHelper;
 export let unregisterWaiter;
 
-if (has('ember-testing')) {
+if (DEBUG) {
   let {
     Test
-  } = require('ember-testing');
+  } = testing;
 
   registerAsyncHelper = Test.registerAsyncHelper;
   registerHelper = Test.registerHelper;
@@ -17,7 +25,7 @@ if (has('ember-testing')) {
   unregisterWaiter = Test.unregisterWaiter;
 } else {
   let testingNotAvailableMessage = () => {
-    throw new Error('Attempted to use test utilities, but `ember-testing` was not included');
+    console.error(new Error("Attempted to use test utilities, but `ember-testing` was not included"));
   };
 
   registerAsyncHelper = testingNotAvailableMessage;
diff --git a/node_modules/ember-source/dist/packages/ember/index.js b/node_modules/ember-source/dist/packages/ember/index.js
index 9382f12..a0dfc65 100644
--- a/node_modules/ember-source/dist/packages/ember/index.js
+++ b/node_modules/ember-source/dist/packages/ember/index.js
@@ -1,4 +1,3 @@
-import require, { has } from 'require';
 import { getENV, getLookup, setLookup } from '@ember/-internals/environment';
 import * as utils from '@ember/-internals/utils';
 import { Registry, Container } from '@ember/-internals/container';
@@ -7,10 +6,11 @@ import { meta } from '@ember/-internals/meta';
 import * as metal from '@ember/-internals/metal';
 import { FEATURES, isEnabled } from '@ember/canary-features';
 import * as EmberDebug from '@ember/debug';
+import { DEBUG } from '@glimmer/env';
 import { assert, captureRenderTree, deprecate } from '@ember/debug';
 import Backburner from 'backburner';
 import Controller, { inject as injectController, ControllerMixin } from '@ember/controller';
-import { _getStrings, _setStrings, dasherize, camelize, capitalize, classify, decamelize, loc, underscore, w } from '@ember/string';
+import { _getStrings, _setStrings, dasherize, camelize, capitalize, classify, decamelize, underscore, w } from '@ember/string';
 import Service, { service } from '@ember/service';
 import EmberObject, { action, computed, observer } from '@ember/object';
 import { dependentKeyCompat } from '@ember/object/compat';
@@ -53,6 +53,39 @@ import { isNone, isBlank, isEmpty, isPresent, isEqual, typeOf, compare } from '@
 import { templateOnlyComponent, invokeHelper, hash, array, concat, get, on, fn } from '@glimmer/runtime';
 import { helperCapabilities, setModifierManager, setComponentTemplate, getComponentTemplate, setHelperManager } from '@glimmer/manager';
 import { assertDestroyablesDestroyed, associateDestroyableChild, destroy, enableDestroyableTracking, isDestroying, isDestroyed, registerDestructor, unregisterDestructor } from '@ember/destroyable'; // ****@ember/-internals/environment****
+import * as testing from 'ember-testing';
+import * as waiters from 'ember-testing/lib/test/waiters'
+
+const registry = {
+  'ember-testing/lib/test/waiters': DEBUG ? waiters : undefined,
+};
+
+function define(a, b) {
+  console.log(`define: ${a}`, b);
+  registry[a] = b;
+}
+
+function require(a) {
+  if (a in registry) {
+    console.log(`require from registry: ${a}`);
+
+    return registry[a];
+  }
+  console.log(`require: ${a}`);
+  try {
+    return window.requireModule(a);
+  } catch(e) {
+    console.error(e);
+  }
+}
+
+function has(a) {
+  if (a === 'ember-testing') {
+    return DEBUG ? true : false;
+  }
+  console.log(`has: ${a}`);
+  return false;
+}
 
 const Ember = {};
 Ember.isNamespace = true;
@@ -219,7 +252,6 @@ Ember._Backburner = Backburner; // ****@ember/-internals/runtime****
 
 Ember.A = A;
 Ember.String = {
-  loc,
   w,
   dasherize,
   decamelize,
@@ -442,9 +474,9 @@ function defineEmberTemplateCompilerLazyLoad(key) {
     enumerable: true,
 
     get() {
-      if (has('ember-template-compiler')) {
+      if (false) {
         let templateCompiler = require('ember-template-compiler');
-
+        console.log(templateCompiler);
         EmberHTMLBars.precompile = EmberHandlebars.precompile = templateCompiler.precompile;
         EmberHTMLBars.compile = EmberHandlebars.compile = templateCompiler.compile;
         Object.defineProperty(Ember, 'HTMLBars', {
@@ -477,9 +509,7 @@ function defineEmberTestingLazyLoad(key) {
     enumerable: true,
 
     get() {
-      if (has('ember-testing')) {
-        let testing = require('ember-testing');
-
+      if (DEBUG) {
         let {
           Test,
           Adapter,
@@ -509,14 +539,17 @@ function defineEmberTestingLazyLoad(key) {
   });
 }
 
+
+
 defineEmberTestingLazyLoad('Test');
 defineEmberTestingLazyLoad('setupForTesting');
 runLoadHooks('Ember');
+
 Ember.__loader = {
   require,
   // eslint-disable-next-line no-undef
   define,
   // eslint-disable-next-line no-undef
-  registry: typeof requirejs !== 'undefined' ? requirejs.entries : require.entries
+  registry: registry
 };
 export default Ember;
\ No newline at end of file
