diff --git a/node_modules/ember-qunit/.DS_Store b/node_modules/ember-qunit/.DS_Store
new file mode 100644
index 0000000..013777b
Binary files /dev/null and b/node_modules/ember-qunit/.DS_Store differ
diff --git a/node_modules/ember-qunit/addon-test-support/adapter.js b/node_modules/ember-qunit/addon-test-support/adapter.js
index 7673f39..072eb93 100644
--- a/node_modules/ember-qunit/addon-test-support/adapter.js
+++ b/node_modules/ember-qunit/addon-test-support/adapter.js
@@ -1,5 +1,6 @@
 import Ember from 'ember';
 import * as QUnit from 'qunit';
+import * as testing from 'ember-testing';
 import hasEmberVersion from '@ember/test-helpers/has-ember-version';
 
 function unhandledRejectionAssertion(current, error) {
@@ -27,7 +28,7 @@ function unhandledRejectionAssertion(current, error) {
 
 export function nonTestDoneCallback() {}
 
-let Adapter = Ember.Test.Adapter.extend({
+let Adapter = testing.Adapter.extend({
   init() {
     this.doneCallbacks = [];
     this.qunit = this.qunit || QUnit;
diff --git a/node_modules/ember-qunit/vendor/.DS_Store b/node_modules/ember-qunit/vendor/.DS_Store
new file mode 100644
index 0000000..2ce5e5a
Binary files /dev/null and b/node_modules/ember-qunit/vendor/.DS_Store differ
